pytest.ini

```diff
@@ -3,4 +3,9 @@ filterwarnings =
 # upstream lib imports ABC improperly for backward compatibility
     ignore::DeprecationWarning:google.protobuf.descriptor
     ignore::DeprecationWarning:google.protobuf.internal.well_known_types
+# Silence: "Your application has authenticated using end user credentials from Google Cloud SDK"
+    ignore::UserWarning:google.auth
 norecursedirs = venv
+markers =
+  black
+  mypy
```

Comment:
the pytest plugins are supposed to register their markers, but it's a new thing and these don't yet and it was causing warnings